tests/pauli/dla/test_lie_closure.py

```diff
@@ -678,3 +678,67 @@ def test_non_hermitian_error(
         ops = [np.array([[0.0, 1.0], [0.0, 0.0]])]
         with pytest.raises(ValueError, match="At least one basis matrix"):
             _ = qml.lie_closure(ops, matrix=True)
+
+
+X0 = qml.matrix(X(0))
+Y0 = qml.matrix(Y(0))
+Z0 = qml.matrix(Z(0))
+
+
+@pytest.mark.all_interfaces
+class TestLieClosureInterfaces:
+    """Test input for matrix inputs from AD interfaces"""
+
+    @pytest.mark.jax
```

Comment:
Doesn't `all_interfaces` already fix in which cases these tests are run? Do we need `mark.jax` etc in addition? :o

Reply:
yeah though the all_interfaces is removed now and just the individual interfaces left

Reply:
Perfect, thanks! :)